### PR TITLE
en.yaml: refine wording and format; unify term usages

### PR DESCRIPTION
**Pull request by yikeke**
My review link is [here](http://172.16.5.143:2379/dashboard/api/diagnose/reports/3600d292-c88c-435c-b433-f6095d17e23a/detail#process_cpu_usage).

### CODE REVIEW

File: ui/diagnoseReportApp/translations/en.yaml [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+      tidb_gc_change_config: TiDB GC Config Change History
       tikv_rocksdb_initial_config: TiKV RocksDB Initial Config
-      tikv_rocksdb_change_config: TiKV RocksDB Change Config
+      tikv_rocksdb_change_config: TiKV RocksDB Config Change History
       tikv_raftstore_initial_config: TiKV RaftStore Initial Config
-      tikv_raftstore_change_config: TiKV RaftStore Change Config
-      pd_time_consume: PD Time Consume
+      tikv_raftstore_change_config: TiKV RaftStore Config Change History
+      pd_time_consume: Time Consumed by PD Component
       blance_leader_region: Blance Leader/Region
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:52]: typo: s/blance/balance?

File: ui/diagnoseReportApp/translations/en.yaml [hunk 2]
```diff
@@ -1,103 +1,103 @@
...
-      tikv_rocksdb_change_config: TiKV RocksDB Change Config
+      tikv_rocksdb_change_config: TiKV RocksDB Config Change History
       tikv_raftstore_initial_config: TiKV RaftStore Initial Config
-      tikv_raftstore_change_config: TiKV RaftStore Change Config
-      pd_time_consume: PD Time Consume
-      blance_leader_region: Blance Leader/Region
-      approximate_region_size: Approximate Region size
+      tikv_raftstore_change_config: TiKV RaftStore Config Change History
+      pd_time_consume: Time Consumed by PD Component
+      blance_leader_region: Scheduled Leader/Region Count
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:52]:
```suggestion
      blance_leader_region: Scheduled Leader/Region Count
```
typo: s/blance/balance?

Comment (breeswish) [reply ui/diagnoseReportApp/translations/en.yaml:52]: @crazycs520

Comment (crazycs520) [reply ui/diagnoseReportApp/translations/en.yaml:52]: yes, should be balance

Comment (crazycs520) [reply ui/diagnoseReportApp/translations/en.yaml:52]: fix in https://github.com/pingcap-incubator/tidb-dashboard/pull/495

**Comment by breeswish** [comment]
Thank you!

**Comment by breeswish** [comment]
@yikeke Is this PR ready to merge now?

**Comment by yikeke** [comment]
> @yikeke Is this PR ready to merge now?

Let me take a last look.

File: ui/diagnoseReportApp/translations/en.yaml [hunk 3]
```diff
@@ -107,197 +107,197 @@ diagnosis:
...
-        tikv_pd_request: The time cost of TiKV sends requests to PD, the label is [type]
+        tidb_query: The time cost of SQL queries. The label is [sql_type].
+        tidb_get_token(us): The time cost of session getting token to execute the SQL query. The label is [instance].
+        tidb_parse: The time cost of parsing SQL queries. The label is [sql_type].
+        tidb_compile: The time cost of building the query plan. The label is [sql_type].
+        tidb_execute: The time cost of executing the SQL query, which does not include the time to get the results of the query. The label is [sql_type].
+        tidb_distsql_execution: The time cost of distsql execution. The label is [type].
+        tidb_cop: The processing time of KV storage Coprocessor. The label is [instance].
+        tidb_transaction: The time cost of durations when executing a transaction, including retry. The label is [sql_type].
+        tidb_transaction_local_latch_wait: The time cost of. The label is [instance].
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:162]: The time cost of what? The sentence is not finished.

File: ui/diagnoseReportApp/translations/en.yaml [hunk 4]
```diff
@@ -107,197 +107,197 @@ diagnosis:
...
+        tidb_kv_backoff: The time cost of TiDB transaction latch wait time on key value storage. The label is [type].
+        tidb_kv_request: The time cost of KV requests durations. The label is [type].
+        tidb_slow_query: The time cost of TiDB slow query. The label is [instance].
+        tidb_slow_query_cop_process: The time cost of TiDB slow query total cop process time. The label is [instance].
+        tidb_slow_query_cop_wait: The time cost of TiDB slow query total cop wait time. The label is [instance].
+        tidb_ddl_handle_job: The time cost of handle TiDB DDL job. The label is [type].
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:168]:
```suggestion
        tidb_ddl_handle_job: The time cost of TiDB DDL handling job. The label is [type].
```

File: ui/diagnoseReportApp/translations/en.yaml [hunk 5]
```diff
@@ -107,197 +107,197 @@ diagnosis:
...
+        tidb_ddl_worker: The time cost of DDL worker handle job. The label is [action].
+        tidb_ddl_update_self_version: The time cost of TiDB schema syncer version update. The label is [result].
+        tidb_owner_handle_syncer: The time cost of TiDB DDL owner operations on etcd. The label is [type].
+        tidb_ddl_batch_add_index: The time cost of TiDB batch add index. The label is [type].
+        tidb_ddl_deploy_syncer: The time cost of TiDB ddl schema syncer statistics, including init, start, watch, and clear. The label is [type].
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:173]:
```suggestion
        tidb_ddl_deploy_syncer: The time cost of TiDB DDL schema syncer statistics, including init, start, watch, and clear. The label is [type].
```

File: ui/diagnoseReportApp/translations/en.yaml [hunk 6]
```diff
@@ -107,197 +107,197 @@ diagnosis:
...
+        tidb_load_schema: The time cost of TiDB loading schema. The label is [type].
+        tidb_meta_operation: The time cost of TiDB meta operations, including get/set schema and ddl jobs. The label is [instance].
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:175]:
```suggestion
        tidb_meta_operation: The time cost of TiDB meta operations, including get/set schema and DDL jobs. The label is [instance].
```

File: ui/diagnoseReportApp/translations/en.yaml [hunk 7]
```diff
@@ -107,197 +107,197 @@ diagnosis:
...
+        tidb_auto_id_request: The time cost of TiDB auto id, handle id requests. The label is [type].
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:176]:
```suggestion
        tidb_auto_id_request: The time cost of handling requests for TiDB auto ID. The label is [type].
```

File: ui/diagnoseReportApp/translations/en.yaml (hunk 3 above)
Comment (breeswish) [reply ui/diagnoseReportApp/translations/en.yaml:162]: The time cost of local latch wait

Comment (yikeke) [reply ui/diagnoseReportApp/translations/en.yaml]: Updated in https://github.com/pingcap-incubator/tidb-dashboard/pull/461/commits/c6d827eabb77e609ea4a83052aaf091c606a00c7

File: ui/diagnoseReportApp/translations/en.yaml [hunk 8]
```diff
@@ -107,197 +107,197 @@ diagnosis:
...
+        tikv_scheduler_scan_details_total_num: The keys scan details of each CF when executing a command. The label is [instance,req,tag].
+        tikv_scheduler_stage_total_num: The total number of scheduler states. The label is [instance,type,stage].
+        tikv_gc_keys_total_num: The total number of keys in CF affected during GC. The label is [instance,cf,tag].
+        tidb_gc_worker_action_total_num: The total count of KV storage garbage collection. The label is [instance,type].
+        tikv_worker_handled_tasks_total_num: The total number of tasks handled by worker. The label is [instance,name].
+        tikv_worker_pending_tasks_total_num: The total number of pending and running tasks of worker. The label is [instance,name].
+        tikv_futurepool_handled_tasks_total_num: The total number of tasks handled by future_pool. The label is [instance,name].
+        tikv_futurepool_pending_tasks_total_num: The total number of pending and running tasks of future_pool. The label is [instance,name].
+        tikv_snapshot_kv_count: tikv_snapshot_kv_count. The label is [instance].
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:245]: There is no explanation for tikv_snapshot_kv_count.

File: ui/diagnoseReportApp/translations/en.yaml [hunk 9]
```diff
@@ -107,197 +107,197 @@ diagnosis:
...
+        tikv_snapshot_size: The number of KV pairs within a snapshot. The label is [instance].
+        tikv_snapshot_state_total_count: tikv_snapshot_size. The label is [instance,type].
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:247]: tikv_snapshot_size is not an appropriate explanation to tikv_snapshot_state_total_count.

File: ui/diagnoseReportApp/translations/en.yaml [hunk 10]
```diff
@@ -107,197 +107,197 @@ diagnosis:
...
+        tidb_lock_resolver_total_num: The total count of lock resolve. The label is [instance].
+        pseudo_estimation_total_count: The total count of TiDB Optimizer using pseudo estimation. The label is [instance, type].
+        dump_feedback_total_count: The total count of operations that TiDB dumping statistics back to KV storage. The label is [instance, type].
+        store_query_feedback_total_count: The total count of TiDB store querying feedback. The label is [instance, type].
+        update_stats_total_count: The total count of TiDB updating statistics using feed back. The label is [instance].
+        balance-leader-in: balance-leader-in is the total count of Leader moving into the TiKV store. The label is [address].
+        balance-leader-out: balance-leader-out is the total count of Leader moving out of the TiKV store. The label is [address].
+        balance-region-in: balance-region-in is the total count of Regions moving into the TiKV store. The label is [address].
+        balance-region-out: balance-region-in is the total count of Regions moving into the TiKV store. The label is [address].
```
Comment (yikeke) [inline ui/diagnoseReportApp/translations/en.yaml:232]: I fixed the typos in the four variables. PTAL.

**Comment by breeswish** [comment]
@yikeke Thanks a lot!!

**Comment by breeswish** [comment]
@all-contributors please add @yikeke   for content

**Pull request merged**